src/Step/Deploy/InstallUpdate/Install/Setup.php

```diff
@@ -70,11 +79,12 @@ public function execute()
 
             $this->shell->execute('echo \'Installation time: \'$(date) | tee -a ' . $installUpgradeLog);
             $this->shell->execute(sprintf(
-                '/bin/bash -c "set -o pipefail; %s | tee -a %s"',
+                '%s -c "set -o pipefail; %s | tee -a %s"',
+                $this->utilityManager->get(UtilityManager::UTILITY_BASH),
```

Comment:
Make sense to rename UTILITY_BASH to UTILITY_SHELL

Reply:
It's `bash` in `which bash` representation. Why would we use `shell`?

Reply:
Fixed